exercises/concept/troll-the-trolls/.meta/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "authors": [
+    "vaeng"
+  ],
+  "files": {
+    "solution": [
+      "troll_the_trolls.cpp"
+    ],
+    "test": [
+      "troll_the_trolls_test.cpp"
+    ],
+    "exemplar": [
+      ".meta/exemplar.cpp"
+    ]
+  },
+  "icon": "attack-of-the-trolls",
+  "blurb": "Learn about enumerations and switch statements by checking permissions of user accounts on an internet forum."
```

Comment:
```suggestion
  "blurb": "Learn about enumerations and switch statements by checking permissions of user accounts."
```

Reply:
This is both for the game and the forum, so I removed the reference to the forum.